app/components/TagContainer.js

```diff
@@ -126,7 +136,19 @@ class TagContainer extends React.Component<Props> {
         role="presentation"
         data-tid={'tagContainer_' + tag.title.replace(/ /g, '_')}
         key={key || tag.id || uuidv1()}
-        onClick={event => { if (this.props.handleTagMenu) { this.props.handleTagMenu(event, tag, entryPath || tagGroup); } }}
+        onClick={event => {
+          if (event.ctrlKey) {
```

Comment:
Hey @IHappyDayI , I finally got time to review your PR...
What is the idea behind this check for holding the ctrl key?

Reply:
Well, the idea is to very quickly add tags by cltr+clicking them. I didn't want to override the default click behavior, which opens the tag menu.

Reply:
Does this worked for your? In this branch https://github.com/tagspaces/tagspaces/tree/taggingFeatures , where I have merged your branch and the current develop, it does nothing...

Reply:
Well, if you run the branch with a debugger attached it tells you addTags is not a function. It looks like you removed the mapping of addTags. removeTags and selectedEntries in that branch.

If you want to, I can take some time during the next few days to rebase this pull request to resolve all conflicts, so it can be merged without issues.

Reply:
I will try to fix the problem by myself. I will write you if I need help. The problems is that addTags is already in TagConainerDnd. I do not wanted to have it imported twice. I want to make the TagContainer as light as possible...